source/html/renderer/src/card-elements.ts

```diff
@@ -470,7 +470,7 @@ export class FactSet extends CardElement {
             return this.speak + '\n';
```

Comment:
You are correct. Let me double-check that isSelected is indeed not the design as the implementation will depend on that. I'll get back to you.